src/components/Button.js

```diff
@@ -1,21 +1,29 @@
+/* eslint-disable react/require-default-props */
```

Comment:
- [ ] OPTIONAL
You have **disabled ESLint checks** on parts of your code. Please enable **all** ESLint checks on your code and try modifying that portion of your code to adhere to ESLint standards. This will ensure a robust and clean code.